Mods/intact_mods.txt

```diff
@@ -19,4 +19,10 @@ TG   L or I
 MT   Unlocalized
 MM   -113.08406397853
 CF   C-6 H-11 O-1 N-1
-//
\ No newline at end of file
+//
+ID   Sulfate Adduct
```

Comment:
I think fairly certain based on that paper. The mass matches. Doing some tests this next week to see if it's from SDS, but for now I think it's good to label so the peak gets picked/identified